Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Dockerfile with running fat jar of Schema Guru Web UI
+
+# Ubuntu build with java 7 (https://index.docker.io/u/dockerfile/java/)
+FROM java:7
+
+MAINTAINER Anton Parkhomenko, mailbox@chuwy.me
```

Comment:
Hey @chuwy, can you change maintainer to Snowplow Analytics, support@snowplowanalytics.com please?